From 56fde2494ab4b737448621a32b04ebd94ea4135f Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Sat, 7 Jul 2018 22:40:59 +0100
Subject: [PATCH 11/15] msm8953-common: Remove implicitly built QTI camera
 shared libraries

 * These shared libs are built when TARGET_USES_QTI_CAMERA_DEVICE
   flag is enabled.

Change-Id: Ifb39b7c2c8de2cbb1f59c6dc492a3763d32f1b7a
---
 msm8953.mk | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/msm8953.mk b/msm8953.mk
index 405f428..fa42c49 100644
--- a/msm8953.mk
+++ b/msm8953.mk
@@ -131,9 +131,7 @@ PRODUCT_PACKAGES += \
     android.hardware.camera.provider@2.4-service \
     camera.msm8953 \
     libmm-qcamera \
-    Snap \
-    vendor.qti.hardware.camera.device@1.0 \
-    vendor.qti.hardware.camera.device@1.0_vendor
+    Snap
 
 # Configstore
 PRODUCT_PACKAGES += \
-- 
2.7.4

